### PR TITLE
SET_CELL_FORMULA: set formula even if cell was uninitialized

### LINKED ISSUE #866: `SET_CELL_FORMULA` doesn't work for cells not previously initialized
The following code doesn't show a formula in cell A1:
```
REPORT.
CONSTANTS: gc_save_file_name TYPE string VALUE 'test.xlsx'.
INCLUDE zdemo_excel_outputopt_incl.
START-OF-SELECTION.
  DATA(lo_excel) = new zcl_excel( ).
  DATA(lo_worksheet) = lo_excel->get_active_worksheet( ).
  lo_worksheet->set_cell_formula( ip_column = 1 ip_row = 1 ip_formula = '"hello"&" "&"world"' ).
  lcl_output=>output( lo_excel ).
```
Actual result:
image

Expected result:
image

### PR DESCRIPTION
**Pull request by sandraros**
Fix #866

### CODE REVIEW

**Comment by larshp** [comment]
The PR title "Fix #866" is the text shown in the PR overview, and typically also what is merged to main if the commits are squashed

suggest having the PR title something like "SET_CELL_FORMULA: fix uninitialized cells"

The issue and PR is linked only by the description of the PR or the comments

**Comment by sandraros** [comment]
@larshp Thanks for telling me. You're right, I will do the way you suggest.

File: src/zcl_excel_worksheet.clas.abap [hunk 1]
```diff
@@ -3724,8 +3724,8 @@ CLASS zcl_excel_worksheet IMPLEMENTATION.
     lv_column = zcl_excel_common=>convert_column2int( ip_column ).
     READ TABLE me->sheet_content ASSIGNING <sheet_content> WITH TABLE KEY cell_row    = ip_row
                                                                           cell_column = lv_column.
-    IF sy-subrc <> 0.                                                                           " Create new entry in sheet_content if necessary
-      CHECK ip_formula IS INITIAL.                                                              " no need to create new entry in sheet_content when no formula is passed
+    IF sy-subrc <> 0.                   " Create new entry in sheet_content if necessary
+      CHECK ip_formula IS NOT INITIAL.  " no need to create new entry in sheet_content when no formula is passed
```
Comment (AndreaBorgia-Abo) [inline src/zcl_excel_worksheet.clas.abap]:
```suggestion
      CHECK ip_formula IS NOT INITIAL.  " only create new entry in sheet_content when a formula is passed
```

Comment (AndreaBorgia-Abo) [reply src/zcl_excel_worksheet.clas.abap]: I suspect the double negative in the comment might be part of the reason why the test was wrong, plus this was as good a chance as any to test the "suggestion" feature.

**Review by AndreaBorgia-Abo — APPROVED** [review]
LGTM

File: src/zcl_excel_worksheet.clas.abap (hunk 1 above)
Comment (sandraros) [reply src/zcl_excel_worksheet.clas.abap]: And me to just "commit suggestion" 😀

**Review by AndreaBorgia-Abo — APPROVED** [review]

**Pull request merged**